charters/Architecture/charter.md

```diff
@@ -0,0 +1,35 @@
+# Architecture++
+
+## Description
+
+Architecture++ is responsible for the technical oversight of the Data Coordination Platform (DCP), including identifying and addressing *cross-cutting* architectural concerns across DCP components.
+
+The core team is composed of *Technical Leads* representing each DCP component to ensure a diversity of skills and views needed for community technical decision-making. 
+
+## In-scope
+Strategic responsibilities
+* Set overall **technical** direction for the DCP
+* Publish an annual Architecture RFC documenting technical debt that needs to be addressed during roadmap planning by DCP PM++
+* Review and approve technical charters for DCP components
+* Review and approve cross-component RFC(s)
```

Comment:
Is it clear to other reviewers that Architecture is the **final** approver for cross-component RFC(s)?